src/main/java/de/wi2020sebgroup1/cinema/entities/Seat.java

```diff
@@ -54,6 +54,20 @@
 	@JoinColumn(name = "show_id", referencedColumnName = "id")
 	private Show show;
 	
+	public Seat() {}
+	
+	public Seat(@NotNull int reihe, @NotNull int place, @NotNull boolean coupleSeat, @NotNull boolean blocked,
```

Comment:
Ein Konstruktor ohne Fremdobjekte wäre gut. Damit man die zumindest mal unabhängig von einander anlegen kann

Reply:
Wofür denn? Ich hatte die leeren nur angelegt, um die in den Tests schön zu benutzen bzw. manchmal gibt es an einigen Stellen vllt nen Use Case dafür - ich hab die aber erstmal rausgenommen.

Reply:
Beim Seat ist eventuell nicht wichtig, allerdings z.B. beim CinemaRoom. Man muss ja entweder erst den SeatingPlan erstellen und dann darauf den SeatingPlan oder andersherum, so kann man aber dank der Abhängigkeiten voneinander nichts erstellen.